src/components/BookItem.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+function BookItem() {
+  return (
+
+    <li>
+      Book1
+      <button type="button">Delete</button>
+    </li>
+
```

Comment:
* I suggest you make the `BookItem` component able to take book data as a prop and render the title and the author this would make it easier to pass book details ( title and author ) HINT 💭 : you could do something like this `function Welcome(props) {
  return <h1>Hello, {props.name}</h1>;
}`

`function App() {
  return (
    <div>
      <Welcome name="Sara" />      <Welcome name="Cahal" />      <Welcome name="Edite" />    </div>
  );
}`